### ISSUE TITLE
Edit folder names

### ISSUE THREAD

**Issue by rocketman768** [Brewtarget/brewtarget#10]
From [launchpad](https://bugs.launchpad.net/brewtarget/+bug/1372542)

Once created the names of recipe folders cannot be changed. They can only be deleted.

**Comment by mikfire** [comment]
Yes. That was by design, actually. I could never quite figure out what renaming a folder should do. 

For example, given a folder /a/b/c/d and you click rename on c. Should you just be able to rename "c", or should you be able to rename "/a/b/c"? I kind of lean toward the first but the dialog always seemed a little weird. On the other hand, being able to right-click "c" and then change the path didn't seem right either.

Maybe it would make more sense if I could figure out how to do an in-line dialog, like what a filesystem browser does when you right click and say "rename".

**Comment by pricelessbrewing** [comment]
+1 for be able to rename just "c" when clicking c in the /a/b/c/d example since that's what most people are most familiar with when renaming folders as part of a directory.

**Comment by mattiasmaahl** [comment]
@mikfire Is this a wanted feature? I can give it a try to see if I can make it happen?

**Comment by matty0ung** [comment]
I have a fix for this running locally and will include it in the 5.1 release.

**Comment by matty0ung** [comment]
Fixed in [5.1.0](https://github.com/Brewtarget/brewtarget/releases/tag/v5.1.0).
